draft-bbs-signatures.md

```diff
@@ -366,21 +379,25 @@ Procedure:
 
 1. (W, H0, H) = octets_to_point(PK)
 
-2. h = XOF(SK  || msg[i] || ... || msg[L])
+2. sig_dst = OS2IP(HASH(PK || L || H_s || H_d || H_1 || ... || H_L || Ciphersuite_ID || CipherInfo)) mod q
+
+3. if sig_dst is 0, abort
 
-3. for rand_el in (e, s) do
+4. h = XOF(SK  || msg[i] || ... || msg[L])
 
-4.      rand_el = OS2IP(h.read(64)) mod q.
+5. for rand_el in (e, s) do
```

Comment:
side note that tripped me up here rand_el implies that the resulting values for e and s are entirely random when they are not should we name this something else?

Reply:
How about `element` or `item` instead?